test/dlc_tests/conftest.py

```diff
@@ -564,9 +553,14 @@ def ec2_instance(
         )
         if ec2_instance_ami != PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_EAST_1:
             ec2_instance_ami = (
-                AML2_BASE_DLAMI_US_EAST_1
-                if ec2_instance_ami == AML2_BASE_DLAMI_US_WEST_2
-                else UBUNTU_20_BASE_DLAMI_US_EAST_1
+                test_utils.get_instance_type_base_dlami(
```

Comment:
Yeah this condition was very weird. I don't quite understand it completely but it seemed like the original commit was trying to cast region `us-east-1` though.

To my understanding, it is trying to force `us-east-1` AML2 AMI if the current `ec2_instance_ami` is an AML2 AMI in `us-west-2`. Otherwise, it uses the UL20 `us-east-1`.